hackage-server.cabal

```diff
@@ -2,6 +2,8 @@ cabal-version: 3.0
 name:         hackage-server
 version:      0.5.1
 
+build-type:   Simple
```

Comment:
If I am not mistaken, `build-type: Simple` is obsolete in `cabal-version 3.0`, since it is the default.

Reply:
you're right, I thought this might make a difference for DocTests in `haskell.nix`, it didn't